examples/contrib/docker-image-2-singularity-container.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+# convert docker images in singularity container
+# requirements:
+# docker and sudo privileges to run docker
+# singularity and sudo privileges to run singularity
+# 2016/11/04 Tru Huynh <tru@pasteur.fr>
+
+# where the containers will be installed
+SINGULARITY_CONTAINER_DIR=~/singularity-img
+
```

Comment:
We should have a more group discussion about where we plan to cache singularity containers, because there are going to be a few types!

- docker layers
- singularity image layers
- (and support for something else)

My suggestion is to have a `~/.singularity` folder, and inside we would have images, with subdirectories for each type. Otherwise, the layers/images are going to get confusing.

This structure would need to be checked for at each run of some kind of bootstrap, and additionally have options for the user to enable or disable the cache, or customize this location. That would be great to add to this PR!